### ISSUE TITLE
Bug: Same Prefixed Method's Docs Conflict

### ISSUE THREAD

**Issue by HasanAshab** [ad-on-is/adonis-autoswagger#63]
My controller has 2 methods **logout** and **logoutOnDevice** with same prefix **logout**.
Thats why AutoSwagger respects the last comment block and use that for both

```typescript
  /**
   * @logout
   * @summary Logout a user
   * @responseBody 200 - { message: "Logged out successfully" }
   */
  async logout({ auth }: HttpContext) {
    await this.authService.logout(auth.user!)
    return 'Logged out successfully!'
  }

  /**
   * @logoutOnDevice
   * @summary Logout a user on specific device
   * @responseBody 200 - { message: "Logged out successfully" }
   */
  async logoutOnDevice({ request, params, auth }: HttpContext) {
    await this.authService.logoutOnDevice(auth.user!, params.id)
    return 'Logged out successfully!'
  }
```

AutoSwagger searchs for **action name** on comments by **includes()**
```typescript
comment.value.includes("@" + action) // src/parsers.ts:471
```
This part causing the bug

**Comment by ad-on-is** [comment]
Would you mind sharing the corresponding route?

**Comment by ad-on-is** [comment]
This should be fixed in 3.2.0
